npm/test/basics.js

```diff
@@ -697,16 +697,6 @@ async function testCompilerError(useWorker) {
 test("compiler error on run", () => testCompilerError(false));
 test("compiler error on run - worker", () => testCompilerError(true));
 
-test("debug service get breakpoints without service loaded returns empty - web worker", async () => {
```

Comment:
I removed this because my changes in the `wasm` crate was causing this to fail now. In practice, I didn't think it made sense to support calling `getBreakpoints` before we ever called `loadSource`. I verified that we never actually do this in the product code.